src/components/TeaserFeed/index.js

```diff
@@ -62,7 +64,7 @@ export const TeaserFeed = ({
       : Headlines.Editorial
 
   return (
-    <Container format={format} color={formatMeta.color || metaColor} Link={Link}>
+    <Container format={format} color={formatColor || formatMeta.color || colors[formatMeta.kind]} Link={Link}>
```

Comment:
Maybe `color={formatMeta.color || (format ? colors[formatMeta.kind] : undefined)}` could work and make the new `formatColor` prop unnecessary?

Reply:
Done, but slightly modified:
`color={(formatMeta.title ? (formatMeta.color || colors[formatMeta.kind]) : undefined)}`